resources/docs/dashboard.yml

```diff
@@ -0,0 +1,59 @@
+openapi: 3.0.0
+info:
+  title: Dashboard Terrafund API
```

Comment:
Since this is part of the existing `v2` API namespace, it seems to make sense to include this documentation in the big swagger doc that already exists. This feels more like an extension of the existing API, rather than a whole new API that needs separate documentation.

Reply:
The intention with having another OpenAPI endpoint only for dashboard is to make separate code generations bases. Otherwise website repository will be generating code for dashboard endpoints which doesn't use and geo repository will be generating code for all the other endpoints which are not going to be use.